packages/elements-react/src/util/clientConfiguration.ts

```diff
@@ -16,21 +16,72 @@ export type OryClientConfiguration = {
 
   stylesheet?: string
 
+  /**
+   * An optional favicon URL to display in the UI.
+   */
   favicon?: string
 
+  /**
+   * The SDK configuration.
+   */
   sdk: {
+    /**
+     * Your SDK URL.
+     *
+     * You can find it here: https://console.ory.sh/projects/current/settings
+     */
     url: string
+
+    /**
+     * Optional SDK configuration parameters.
+     */
     options?: Partial<ConfigurationParameters>
   }
 
-  project: {
-    registration_enabled: boolean
-    verification_enabled: boolean
-    recovery_enabled: boolean
+  /**
+   * Enable or disable registration.
+   *
+   * Defaults to true.
+   */
+  registration_enabled?: boolean
```

Comment:
These should all be required, because otherwise we have another layer of defaults in the Ory Network stack. 

One possible improvement is, to make it possible for the library to "auto discover" these values from the project's API endpoints, so that you don't need to pass them in.

Reply:
Hm, I see. But then the user has to keep those in sync? So I assume we need an endpoint for this?

Reply:
We already have an endpoint for it, but it's internal only for now. 

We can expose it, though, and that was the plan. Just need to make sure it doesn't expose anything.

Reply:
Ok, I think we should solve this for the release, otherwise it's really cumbersome to set up. And then it should be removed from the config in my view as we would just use the SDK URL to fetch the data.

Reply:
>Ok, I think we should solve this for the release, otherwise it's really cumbersome to set up.

But it's just a one time setup, where we can provide a copy & pasteable snippet. I don't think it's that critical. 

And we should still provide the option to override all of these settings manually, for local dev or feature flags.